### ISSUE TITLE
Cannot start GPU instances - Exception on: com.mysql.cj.jdbc.ClientPreparedStatement

### ISSUE THREAD

**Issue by Revellski** [apache/cloudstack#9483]
##### ISSUE TYPE

 * Bug Report
 

##### COMPONENT NAME

~~~
CS core - GPU
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.0
~~~

##### OS / ENVIRONMENT

~~~
XCP-ng 8.2.1 (platform_version: 3.2.1)
MySQL v8.4.2
Ubuntu 24.04 LTS
~~~

##### STEPS TO REPRODUCE

~~~
1. Add a GPU-enabled host
2. Configure ServiceOffering using the GPU family and vGPU type as `passthrough`
3. Try to deploy an instance using the GPU offering

~~~

##### EXPECTED RESULTS

~~~
The instance should start.
~~~

##### ACTUAL RESULTS

The instance fails to start:
```
Exception on: com.mysql.cj.jdbc.ClientPreparedStatement: 

SELECT host_gpu_groups.id, host_gpu_groups.group_name, host_gpu_groups.host_id 
FROM host_gpu_groups INNER JOIN vgpu_types groupId 
ON host_gpu_groups.id=groupId.gpu_group_id 
WHERE host_gpu_groups.host_id = 20 
  AND host_gpu_groups.group_name = x'<REDACTED>' 
  AND (groupId.vgpu_type = x'<REDACTED>' 
  AND groupId.remaining_capacity > 0 ) 
ORDER BY vgpu_types.remaining_capacity DESC ]
```
The ORDER_BY clause should reference the alias `groupId` but not `vgpu_types.remaining_capacity`

**Comment by Revellski** [comment]
This is applicable to KVM hypervisor as well. Switching between the different Deployment planners did not make a difference.

**Comment by nvazquez** [comment]
Hi @Revellski can you please test the PR #9573?

**Comment by DaanHoogland** [comment]
fixed in #9573
